frontend/src/containers/App.js

```diff
@@ -0,0 +1,194 @@
+import React, { useEffect, useRef } from 'react';
+
+import * as rmsClient from '../api/RoomzMediaServerClient.js'
+import './App.css';
+
+
+
+function App(props) {
+  const room_id = useRef();
+  const user_id = useRef();
+  const egress_media = useRef();
+
+  var ICE_SERVERS = [
+    {urls:"stun:stun.l.google.com:19302"}
+  ];
+  var roomyPcs = {};
+  let videos = [
+    { peer_id: "0-0" },
+  ];
+  let videoRefs = [];
+  var myPeerId = "";
+  let egressMediaStream = null;
```

Comment:
Is there a reason for the inconsistent usage of `let` and `var`? I typically just use `let` everywhere since its block-scoping is more intuitive and prevention of redeclaration is helpful. Unless there is a use-case for `var` here?

Reply:
nah I just have split-personality disorder